### DISCUSSION TITLE
Inquiry Regarding The Topics Of Attribution And Donation

### DISCUSSION THREAD

**Discussion by dylanhood** [anuraghazra/github-readme-stats#2604] [Q&A] [answered]
How, exactly, am I supposed to attribute the author(s) of this repository? Am I allowed to add the attribution(s) to a file in my repository called "Attributions.txt"?

If I donate, should I still send the donation to the linked India COVID effort? If I donate does that mean I no longer have to add an attribution?

**Comment by rickstaa** [comment, marked as answer]
Hey @dylanhood. Thanks for wanting to attribute our repository. The main way to attribute to this repository is to make sure your card links to the upstream:

```md
[![Dylans' GitHub stats](https://github-readme-stats.vercel.app/api?username=dylanhood)](https://github.com/anuraghazra/github-readme-stats)
```

This way, people that want to know how you created the card can simply click the card. That having said, this is an OS project under the [MIT Licence](https://github.com/anuraghazra/github-readme-stats/blob/master/LICENSE). You are therefore free to use the cards without attributing to the upstream. 

Also thanks for considering donating to this repository 🙏🏻. This again is optional. You can find donation links for the repository in the sidebar, or you can donate to any of the collaborators directly via their GitHub profile. The [Give.do India COVID campaign](https://give.do/) has ended, but there are many good causes on the site to whom you can donate that need it more than we do. I will ask @anuraghazra whether we can update the README to make this behaviour more clear.
